src/ceph-volume/ceph_volume/devices/lvm/activate.py

```diff
@@ -151,7 +151,7 @@ def activate_bluestore(lvs, no_systemd=False):
     process.run([
         'ceph-bluestore-tool', '--cluster=%s' % conf.cluster,
         'prime-osd-dir', '--dev', osd_lv_path,
-        '--path', osd_path])
+        '--path', osd_path, '--no-mon-config'])
```

Comment:
@liewegas we somehow missed this, but I recall you mentioned avoiding `--no-mon-config` when calling `mkfs`. Is this fine for priming the OSD dir here? seems OK to me.

Reply:
yep!